play/dbt-materialize/dbt/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
```

Comment:
Playing around with import paths is a feature I'd like to see used sparingly. Is this necessary for this package?

Reply:
This seems unnecessary! (Meaning, I can remove these imports and still pass our tests/build our demo project). Removing.